source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst

```diff
@@ -0,0 +1,116 @@
+Linux Distribution          Supported Versions
+========================    ========================
+Alma Linux                  8, 9
+Red Hat Enterprise Linux    8, 9
+Rocky Linux                 8, 9
+Ubuntu                      22.04 LTS
+========================    ========================
+
+
+Importing Windows guest VMs from VMware requires installing the virtio drivers on the hypervisor hosts for the virt-v2v conversion.
+
+On RHEL9 hosts:
+
+    ::
+
+        yum -y install http://jenkins.openvm.eu/files/virtio-win/virtio-win-1.9.33-0.el9_2.noarch.rpm
+
+
+For other Linux distributions:
+
+    ::
+
+        yum -y install https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/stable-virtio/virtio-win.noarch.rpm
+
+Usage
+-----
+
+In the UI, Virtual Machines to import from VMware are listed in *Tools > Import-Export Instances* section, selecting:
+
+.. cssclass:: table-striped table-bordered table-hover
+
+==================== ========================
+Source               Destination Hypervisor  
+==================== ========================
+Migrate From VMware  KVM
+==================== ========================
+
+|import-vm-from-vmware-to-kvm.png|
+
+Selecting the Destination cluster
+---------------------------------
+
+CloudStack administrators must select a KVM cluster to import the VMware Virtual Machines (left side of the image above). Once a KVM cluster is selected, the VMware Datacenter selection part is displayed (right side of the image above).
+
+Selecting the VM from a VMware Datacenter
+-----------------------------------------
+
+CloudStack administrators must select the Source VMware Datacenter:
+
+    - Existing: The existing zones are listed, and for each zone CloudStack will list if there is any VMware Datacenter associated to it. In case it is, it can be selected
+    - External: CloudStack allows listing Virtual Machines from a VMware Datacenter that is not associated to any CloudStack zone. To do so, it needs the vCenter IP address, the datacenter name, and username and password credentials to log in the vCenter.
+
+Once the VMware Datacenter is selected, click on List VMware Instances to display the list of Virtual Machines on the Datacenter
+
+
+Converting and importing a VMware VM
+------------------------------------
+
+.. note:: CloudStack allows importing Running Linux Virtual Machines, but it is recommended that the Virtual Machine to import is powered off and has been gracefully shutdown before the process starts. For Windows Virtual Machines, it is not possible to import them while running.
+
+When importing a Virtual Machine from VMware to KVM, CloudStack performs the following actions:
+
+    - Cloning the Source Virtual Machine on the selected VMware Datacenter: The source Virtual Machine will be cloned in the original state (running or stopped for Linux VMs, or stopped for Windows VMs). The recommended state is the stopped state to prevent inconsistencies when cloning the virtual machine.
```

Comment:
```suggestion
    - Cloning the Source Virtual Machine on the selected VMware Datacenter: The source Virtual Machine will be cloned in the original state (running or stopped for Linux VMs, or stopped for Windows VMs). The recommended state is the stopped state to prevent data inconsistencies or loss when cloning the virtual machine.
```